tensorcircuit/applications/optimization.py

```diff
@@ -0,0 +1,328 @@
+"""
+modules for QUBO problems in QAOA
+"""
+
+from typing import List, Callable, Any
+
+import tensorcircuit as tc
+import numpy as np
+from functools import partial
+import tensorflow as tf
+import scipy.optimize as optimize
+
+from ..cons import backend
+from ..templates.blocks import QAOA_ansatz_for_Ising
+from ..templates.conversions import QUBO_to_Ising
+from tensorflow.python.ops.numpy_ops import np_config
```

Comment:
this should be in the second import group, no comment on group order for later files